components/Textarea/Textarea.tsx

```diff
@@ -116,6 +115,12 @@ const StyledTextarea = styled('textarea', {
       horizontal: {
         resize: 'horizontal'
       }
+    },
+    endAdornment: {
+      true: {
+        paddingBottom: '$5',
+        paddingRight: '$5',
```

Comment:
:information_source: increase padding when there's an end adornment to make sure it doesn't float on top of textarea content